packages/apollo/src/git.ts

```diff
@@ -48,7 +48,8 @@ export const gitInfo = async (): Promise<GitContext | undefined> => {
 
   let committer;
   let branch = ciBranch || prBranch;
-  let remoteUrl = slug;
+  // BUILD_REPOSITORY_ID is for azure pipelines
```

Comment:
This doesn't seem to be right.

When using `apollo service:push` from an Azure repo on my local machine, the git path is correctly shown in the Apollo Engine schema history interface.  When using `apollo service:push` from an Azure pipeline, the following error is shown in the Apollo Engine schema history, with a yellow triangle:

```
The git context submitted to Engine has an invalid remoteUrl of "befddf0a-9669-46cd-af07-aba537276ebe"
```

The [Azure pipeline variables docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#build-variables) also don't mention a `Build.Repository.Id`, but do mention `Build.Repository.Uri`, which would be useful for a remoteUrl.

Reply:
@ngbrown the format for remoteUrl seems to be different, depending on where in that file you're looking. For example, where you highlighted, it's something in the format of (`apollographql/apollo-tooling`). Later on in the file though, it's redefined with the actual remote url. That seems a bit odd. I'll look into  it and see if using the URL is correct. I wouldn't want to break anything that works currently

Reply:
I opened an issue #1064 to track this until I find the answer, since it's a bit easier than doing it here :)